CHANGELOG.md

```diff
@@ -1,4 +1,15 @@
-## 2.9.0 (Unreleased)
+## 2.8.1 (Unreleased)
+
+IMPROVEMENTS:
+* `monitor`: Add monitor `force_delete` parameter ([#535](https://github.com/terraform-providers/terraform-provider-datadog/pull/535)) Thanks [@ykyr](https://github.com/ykyr)
```

Comment:
This applies to all, but should we use the full resource name for consistency, e.g. `datadog_monitor`